docs/assets.rst

```diff
@@ -15,15 +15,19 @@ For example,
 .. code-block:: python
 
     senderBalance = Balance(Int(0)) # get the balance of Txn.accounts[0] (the sender)
+    senderBalanceDirectRef = Balance(Txn.sender()) # get the balance of the sender by passing the account address (bytes)
```

Comment:
To simplify the docs, I think you should modify all the examples to only use direct references. Users can still view the generated API documentation for the functions to see that indexes are also acceptable.

Reply:
Document updated.